docs/standard/library-guidance/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Open-source library guidance
+description: Best practice recommendations for developers to create high quality .NET libraries.
+author: jamesnk
+ms.author: James.NewtonKing
+ms.date: 09/20/2018
+---
+# Open-source library guidance
+
+This guidance provides recommendations for developers to create high-quality .NET libraries. This documentation focuses on the *what* and the *why* when building a .NET library, not the *how*.
+
+Aspects of high-quality open-source .NET libraries:
```

Comment:
This has a lot of advice for improving Technical aspects but not things like, "Well documented!", "Has an organized Issue Tracker", "A great readme" - might be good too!

Reply:
Yes, that's on the todo list. We're starting with technical stability and quality.

Non-functional aspects of an open-source project will be content to add in the future.